README.md

```diff
@@ -96,6 +96,34 @@ Foreman's database. Providers:
 * `rest_v3` provider uses API v2 with Ruby HTTP library, OAuth and JSON (default)
 * `rest_v2` provider uses API v2 with apipie-bindings and OAuth
 
+## Foreman ENC via hiera
+
+There is a function `foreman_enc` to retrieve the ENC data. This returns the
```

Comment:
```suggestion
There is a function `foreman::enc` to retrieve the ENC data. This returns the
```

Reply:
Updated. Leftover from before I namespaced it.